ApiDocs.Validation/MethodDefinition.cs

```diff
@@ -165,13 +172,13 @@ public void AddTestParams(string rawContent)
         /// <param name="documents"></param>
         /// <param name="account"></param>
         /// <returns></returns>
-        public async Task<ValidationResult<HttpRequest>> GenerateMethodRequestAsync(ScenarioDefinition scenario, DocSet documents, IServiceAccount account)
+        public async Task<ValidationResult<HttpRequest>> GenerateMethodRequestAsync(ScenarioDefinition scenario, DocSet documents, IServiceAccount[] account)
```

Comment:
Will account ever have more than 2 accounts (primary and secondary)?
If at most there are the primary and secondary accounts, I would prefer that those accounts be named explicitly.  It makes the code cleaner.

Reply:
example: `IServiceAccount primaryAccount, IServiceAccount secondaryAccount`

Reply:
For now, I don't think we'll ever need more than two accounts. Sounds good, I'll change that